components/inbound-endpoints/org.wso2.carbon.inbound.endpoint/src/main/java/org/wso2/carbon/inbound/endpoint/common/InboundRunner.java

```diff
@@ -74,20 +73,6 @@ public void run() {
         // Wait for the clustering configuration to be loaded.
         while (!init) {
```

Comment:
do we still need this loop? seems we are doing nothing here.

Reply:
This is the other use-case of carbon.mediation.clustering similar to the previous one. Will remove the loop. Thank you for the valuable input.